smartsim/_core/launcher/local/local.py

```diff
@@ -127,11 +139,51 @@ def stop(self, step_name: str) -> UnmanagedStepInfo:
         """
         # step_id is task_id for local. Naming for consistency
         step_id = self.step_mapping[step_name].task_id
-        
+
         self.task_manager.remove_task(str(step_id))
         _, rc, out, err = self.task_manager.get_task_update(str(step_id))
         step_info = UnmanagedStepInfo("Cancelled", rc, out, err)
         return step_info
 
     def __str__(self) -> str:
         return "Local"
+
+    @staticmethod
+    def get_proxy_cmd(step: Step) -> t.List[str]:
+        """Executes a step indirectly through a proxy process. This ensures unmanaged tasks
+        continue telemetry logging after a driver process exits or fails.
+
+        :param step: the step to produce a proxied command for
+        :type step: Step
+        :return: CLI arguments to execute the step via the proxy step executor
+        :rtype: t.List[str]
+        """
+
+        proxy_module = "smartsim._core.entrypoints.indirect"
+        etype = step.meta["entity_type"]
+        status_dir = step.meta["status_dir"]
+        cmd_list = step.get_launch_cmd()
+        encoded_cmd = encode_cmd(cmd_list)
+
+        out, err = step.get_output_files()
+
+        # note: this is NOT safe. should either 1) sign cmd and verify OR 2) serialize step and let
+        # the indirect entrypoint rebuild the cmd... for now, test away...
+        proxied_cmd = [
+            sys.executable,
+            "-m",
+            proxy_module,
+            "+command",
+            encoded_cmd,
+            "+entity_type",
+            etype,
+            "+telemetry_dir",
+            status_dir,
+            "+working_dir",
+            step.cwd,
+            "+output_file",
+            out,
+            "+error_file",
+            err,
```

Comment:
TY for the long args!! Much more readable here IMO!!

Reply:
w00t!